### ISSUE TITLE
[Spike] Search results are overlapping when you search by location

### ISSUE THREAD

**Issue by hallihocine** [department-of-veterans-affairs/va.gov-team#39888]
Perform some analysis/investigations to determine why the results are overlapping when you search by location tab. 
Example: as you see in the screenshot, when I search using my location, the search results showing 58 results on the list and only 19 results are on the Map. (All of the results from 14 to 52 which are present in the list are not displayed in the map).
![Screen Shot 2022-04-12 at 10.55.22 AM.png](https://images.zenhubusercontent.com/60e7390c5052ef146bcc412e/df550b30-3e34-4915-aa3d-623e2ea9e0e1)

**Comment by PJart** [comment]
The issue where we can geocode the address but the Institution has the same address is less prevalent. When it does happen its usually only on 2-5 Institutions. We can flag those in the database as well and maybe do something on that map (maybe a popup message?) to let the user know multiple Institutions are at the same location.

**Comment by PJart** [comment]
Here is a list of the injstiutions by city that have overlapping records and the number of them in each. [https://app.zenhub.com/files/133843125/a3c68b82-0610-4542-a565-6e86e2228361/download](https://app.zenhub.com/files/133843125/a3c68b82-0610-4542-a565-6e86e2228361/download)

**Comment by af-manish** [comment]
@PJart  Please timebox this to completed 4/27/2022 EOD.

**Comment by PJart** [comment]
I was looking at that the data, one thing I did notice was that my script is not checking the address_1 field, that would fix some cases. I did a little more digging and in the script that geocodes, we could add a step where if the latitude and longitude can not be determined from all of the other address fields, it will take the address field on the Institution and will split it after the part of the address that has Rd, Dr, St etc (could use a regular expression on Institution.address.split("[/regex/"]) and example "3423 PIEDMONT RD NE SUITE 485, ATLANTA, GA, 30305, USA" will not geocode, "3423 PIEDMONT RD, ATLANTA, GA, 30305, USA"  will. That will fix a good amount of the issue cases we found and for the ones that can not be determined, we would need to find a better address and highlight that multiple institutions are in one location on the map.
